packages/amplify-category-storage/src/index.ts

```diff
@@ -17,11 +17,10 @@ import {
   headlessAddStorage,
   headlessImportStorage,
   headlessRemoveStorage,
-  headlessUpdateStorage
+  headlessUpdateStorage,
 } from './provider-utils/awscloudformation/storage-configuration-helpers';
 export { categoryName as category } from './constants';
-export { AmplifyDDBResourceTemplate } from './provider-utils/awscloudformation/cdk-stack-builder/types';
-
+export { AmplifyDDBResourceTemplate, AmplifyS3ResourceTemplate } from './provider-utils/awscloudformation/cdk-stack-builder/types';
 
 async function add(context: any, providerName: any, service: any) {
```

Comment:
We still need to add the changes back in from master: https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-category-storage/src/index.ts

Reply:
didn't the rebase cover the changes tied to storage? What are we missing?

Reply:
Part of the refactor I did as part of --headless for S3 was overwritten by the DDB ext changes (only this index.ts file I believe). Since the overwritten changes are included in the commit, rebasing did not fix it.

Reply:
Yup, those changed are on my radar and will be putting up the changes for that shortly. Thanks for bringing that up again.